go/vm/geth/geth.go

```diff
@@ -137,9 +140,9 @@ func createGethInterpreterContext(parameters vm.Parameters) (*geth.EVM, *geth.Co
 	evm.Context.BlockNumber = big.NewInt(context.BlockNumber)
 	evm.Context.Coinbase = common.Address(context.Coinbase)
 	evm.Context.Difficulty = new(big.Int).SetBytes(context.PrevRandao[:])
-	evm.Context.Time = new(big.Int).SetUint64(uint64(context.Timestamp))
+	evm.Context.Time = uint64(context.Timestamp)
 
-	value := new(big.Int).SetBytes(parameters.Value[:])
+	value := uint256.NewInt(0).SetBytes32(parameters.Value[:])
```

Comment:
This could benefit from offering a `ToUint256()` member function for the `Value` type.

Reply:
I moved the `Uint256ToValue` and `ValueToUint256` functions to the `vm/utils.go` file and reused them here.